src/scrapers/movie/IMDB.cpp

```diff
@@ -105,6 +105,7 @@ QString IMDB::defaultLanguageKey()
 void IMDB::search(QString searchStr)
 {
     QString encodedSearch = QUrl::toPercentEncoding(searchStr);
+    QString includeAdult = (Settings::instance()->showAdultScrapers()) ? "true" : "false";
```

Comment:
Better use a bool. There's not need to store it as a string. :-)

Reply:
Thank you for your bool suggestion. Tested and included.